packages/charts/src/specs/settings.tsx

```diff
@@ -140,8 +140,20 @@ export type ElementClickListener = (
 export type ElementOverListener = (
   elements: Array<XYChartElementEvent | PartitionElementEvent | HeatmapElementEvent | WordCloudElementEvent>,
 ) => void;
+
+/** @public */
+export type BrushArea = XYBrushArea | HeatmapBrushEvent;
```

Comment:
nit: I would keep this consistent and `___event` is most fitting. Also since this is breaking we should change `XYBrushArea` to `XYBrushEvent`.

```suggestion
export type BrushEvent = XYBrushEvent | HeatmapBrushEvent;
```

Reply:
Good naming consistency! Thanks!

Reply:
6b4a4b9